content/connect/index.textile

```diff
@@ -0,0 +1,263 @@
+
+h3(#connection-ids). Connection IDs
+
+An active connection ID is guaranteed to be unique in the Ably system whilst it is active, i.e. no other connection will share that connection ID. However, Ably reserves the right to generate a new connection ID later that may be the same as a previously discarded connection ID (once the connection is closed). Therefore we advise customers to not use the connection ID as a perpetual unique identifier as it is possible that a connection ID may be used many times.
+
+h2(#heartbeats). Heartbeats
+
+Heartbeats enable Ably to identify clients that abruptly disconnect from the service, such as where an internet connection drops out or a client changes networks.
+
+Ably sends a heartbeat to connected clients every 15 seconds. If a client goes more than 25 seconds without seeing any server activity from Ably, it assumes that something has gone wrong with the connection and the "connection state":/connect/states will become @disconnected@. The 25 seconds the client waits is the heartbeat interval plus a 10 second margin of error to allow for network delays.
+
+Ably also uses this mechanism to detect dropped client connections, though some details vary depending on the transport used.
+
+It is important to note that this mechanism is only used when something disrupts communication and does not properly terminate the TCP connection. It isn't used when a connection is deliberately closed or disconnected, for example by calling the "@close()@ method":/api/realtime-sdk/connection#close or being disconnected by the server.
+
+The 15 second interval between heartbeats is used to strike a balance between optimizing battery usage for client devices and the time it takes to identify a dropped or unstable connection.
+
+The interval between heartbeats can be customized if your app requires increased battery preservation or to identify dropped connections more quickly. Set a value between 5000 and 1800000 milliseconds (5 seconds and 30 minutes) using the @heartbeatInterval@ parameter within the @transportParams@ property of the "@clientOptions@":/api/realtime-sdk#client-options object.
+
+Using a higher @heartbeatInterval@ can increase the time taken for the Ably service and the client itself to identify a connection has dropped when an abrupt disconnect occurs. The number of "peak connections":/account/LINK-ME may also appear higher as it can take longer to terminate dropped connections. Although @heartbeatInterval@ can be set as high as 30 minutes, Ably does not recommend setting it this high.
+
+The following example code demonstrates establishing a connection to Ably with a @heartbeatInterval@ of 10 seconds:
+
+bc[jsall]. const ably = new Ably.Realtime(
+  {
+    key: '{{API_KEY}}',
+    transportParams: { heartbeatInterval: 10000 }
+  }
+);
+
+h2(#closing-a-connection). Closing a connection
+
+p(note). There is a 2 minute delay if a connection is not explicitly closed which could impact peak connections and thus costs.
```

Comment:
This doesn't need to be in a note and should form part of the content. Could do with fleshing out the implications listed here too.